src/frontend/render/settings.rs

```diff
@@ -177,6 +177,10 @@ fn format_tui_settings(app: &TuiApp) -> Vec<SettingsItem> {
                 .max_addrs
                 .map_or_else(|| String::from("auto"), |m| m.to_string()),
         ),
+        SettingsItem::new(
+            "tui-custom-columns",
+            format!("{:?}", app.tui_config.tui_columns),
```

Comment:
Currently renders as:

```
Columns([Ttl, Host, LossPct, Sent, Received, Last, Average, Best, Worst, StdDev,
```

(gets truncated in the TUI)

Maybe just show in the `HOLSRAVBWDT` form?

Reply:
As mentioned above I will be changing to display HOLSRAVBWDT.

Reply:
Turned out to be lowercase when displayed which might be preferred by users when setting columns.  Who would hit shift unless they really had to ;)